style.css

```diff
@@ -264,16 +270,230 @@ body {
   padding-bottom: 15px;
   padding-top: 15px;
   border-radius: 10px;
-  margin-right: 20px;
+  gap: 10px;
   display: flex;
 }
 
-.m-l {
-  margin-left: 20px;
-}
-
 .language {
   display: flex;
   justify-content: space-between;
   padding-right: 60px;
 }
+
+@media only screen and (min-width: 768px) {
+  .section-a-container {
+    background-image: url("pimages/header-background-mobile.png");
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
+
+  .navbar-content img {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav_link {
+    display: block;
+  }
+
+  .btn {
+    margin-right: -9px;
+  }
+
+  .nav_link ul {
+    display: flex;
+    list-style: none;
+    margin-top: 50px;
+    margin-right: 50px;
+  }
+
+  .nav_link ul a {
+    text-decoration: none;
+    color: black;
+    padding: 10px;
+    font-size: 15px;
+  }
+
+  .desktop {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-content: center;
+    background-color: #f7f7f9;
+    padding-bottom: 5%;
+  }
+
+  .container-b img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .container-b {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 2%;
+    padding: 1rem 0;
+    border-radius: 24px;
+  }
+
+  .container-b > * {
+    flex-basis: calc(50% - 20px);
+  }
+
+  .ruby {
+    padding: 4px 20px;
+    margin-right: -1px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    width: max-content;
+  }
+
+  .content ul {
+    width: max-content;
+    font-size: 18px;
+    color: #6b778c;
+  }
+
+  .content p {
+    /* margin-top: -20px; */
+    font-size: 16px;
+  }
+
+  .container-c {
+    display: grid;
+    grid-template-columns: 1.4fr 2fr;
+  }
+
+  .about-div {
+    padding-left: 50px;
+  }
```

Comment:
Kindly update your about section to match the figma design better🙏

Your design:
<img width="507" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/20c3b1af-9ca0-4b6d-b804-c36a92cbcb87">

Figma design:
<img width="493" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/e6297345-8f5d-4f59-a2f3-09b34416700e">

A few things to note:
- Please increase the spacing between the elements on the left-hand side
- Please update the color of the `let's connect` heading
- Please increase the spacing between the technologies (HTML, CSS, JS)